[fmem] add package

Ambrou opened the pull request:
Hi,

Please find in this pull request to integrate the fmem Library
The code comes from https://github.com/c-libs/fmem

Ambrou committed:
Initialize project

Ambrou committed:
Initialize project

Ambrou committed:
Initialize project

Ambrou committed:
Rajout de fmem de Ambrou en version 1.0

Ambrou committed:
Rajout de fmem de Ambrou en version 1.0

Ambrou committed:
Rajout de fmem de Ambrou en version 1.0

Ambrou committed:
Rajout de fmem de Ambrou en version 1.0

Ambrou committed:
Mise en place de fmem

Ambrou committed:
Mise en place de fmem

Ambrou committed:
Mise en place de fmem

Ambrou committed:
Mise en place de fmem

cenit committed:
[ogre] update to v1.10.11

cenit committed:
[OpenCV] update to 3.4.1

jclay committed:
[cgal] update hash to 4.11.1

ras0219-msft committed:
[vcpkg] Enable qualifiers to use ! for inversion

ras0219-msft committed:
[vcpkg] Improve handling of external toolchains. Extract MSVC settings to a separate toolchain file.

ras0219-msft committed:
[vcpkg] Improve defaults for other platforms

jasjuang committed:
[alembic] update to 1.7.6

Commit:
Use original reposotory

Commit:
Use original reposotory

Commit:
Use original reposotory

Commit:
Use original reposotory

Commit:
Cleanup cmake file

Commit:
Cleanup cmake file

Commit:
Cleanup cmake file

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=2989) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=2989) before we can accept your contribution.<br/><br/>:x: Ambrou [sign now](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=2989)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/vcpkg?pullRequest=2989) it.</sub>

ras0219-msft committed:
[msix] Initial commit of 1.0

MathiasMagnus committed:
Add package clBLAS (#2944)

* v2.12.2 ships outdated FindOpenCL.cmake

* clBLAS installs, passes checks, usable by 3rd party

* Added usage file

* [clblas] Convert to using a patch file

* [clfft] Revert removal of FindOpenCL -- this can be done, but it will require additional source patching

ras0219-msft committed:
[rttr] Fix executables. Fix read-only permissions.

ras0219-msft committed:
[rocksdb] Disable optional dependencies to improve reliability.

janisozaur committed:
[libzip] Update with latest patches (#2990)

ras0219-msft committed:
[libxml2] Fix vcpkg toolchain helper

ras0219-msft committed:
[vcpkg] Use unzip/zip on non-windows

alexkaratarakis committed:
Merge pull request #2962 from jclay/master

[cgal] update hash for 4.11.1

alexkaratarakis committed:
Merge pull request #2987 from jasjuang/alembic

[alembic] update to 1.7.6

alexkaratarakis committed:
Merge pull request #2932 from cenit/dev/cenit/ogre_upd

[ogre] update to v1.10.11

ras0219-msft committed:
[cgal] Use commit SHA to ensure a stable reference

ras0219-msft committed:
[vcpkg] Fix issue in upgrade dependency resolution where self-referencing features cause cycles

Commit:
Add gherkin-c port in vcpkg Microsoft tool

Commit:
Add gherkin-c port in vcpkg Microsoft tool

Commit:
Add gherkin-c port in vcpkg Microsoft tool

Commit:
Add gherkin-c port in vcpkg Microsoft tool

ras0219-msft committed:
[ogre] Bump control version

reyzal2018 committed:
Add port quirc (#2949)

* fixed SHA512 sum of the package realsense2

* Add port quirc for #2899

* Add port FastCDR for #2953

kichristensen committed:
Update OpenExr to 2.2.1 (#2931)

Commit:
Use original reposotory

Commit:
Use original reposotory

Commit:
Use original reposotory

Commit:
Use original reposotory

Commit:
Merge branch 'master' of https://github.com/Ambrou/vcpkg

ras0219-msft committed:
[vcpkg] Add VCPKG_DEFAULT_VS_PATH environment variable

ras0219-msft committed:
[cimg][doctest][harfbuzz][libwebsockets] Upgrades

horenmar committed:
Update Catch to 2.2.1 (#3004)

wimokn committed:
[uwebsockets] Update to 0.14.6 (#3001)

ras0219-msft committed:
[clara][jsonnet][rs-core-lib][thrift] Upgrades

ras0219-msft committed:
[boost-build] Split out into boost-modular-build-helper and support non-Windows targets

ras0219-msft committed:
[vcpkg] Fix extracting on Windows Server Core. Related to #2518.

ras0219-msft committed:
[vcpkg] Enable packages to provide cmake wrapper scripts.

ras0219-msft committed:
[vcpkg-fixup-cmake-targets] Remove stray absolute references inside top-level cmake files.

ras0219-msft committed:
[vcpkg] Don't set CMAKE_CROSSCOMPILING when targetting Linux-on-Linux

ras0219-msft committed:
[many ports] Improve behavior on Linux and general cleanup

ras0219-msft committed:
[evpp] Fix build and use external rapidjson

ras0219-msft committed:
[evpp] Upgrade to 0.7.0

ras0219-msft committed:
[libzip] Upgrade to 1.5.0

yurybura committed:
[rocksdb] Fix build and use debug suffix 'd'
[gtest] Fix issue #2373

Commit:
[WTL] Update to 10.0.7336

yurybura committed:
[gtest] Fix issue #2373

yurybura committed:
Update versions

yurybura committed:
Revert version

alexkaratarakis committed:
Merge pull request #3015 from gouranga/wtl10_7336

[WTL] Update to 10.0.7336

alexkaratarakis committed:
Merge pull request #3013 from yurybura/master

[rocksdb] Fix build and use debug suffix 'd'

ras0219-msft committed:
[abseil][breakpad][cimg][matio][piex][rocksdb][thrift][tiny-dnn][unicorn-lib][unicorn] Updates

ras0219-msft committed:
[matio][rocksdb] Fix build

ras0219-msft committed:
[vcpkg] Refactor do_build_package and add archive tombstoning

ras0219-msft committed:
[libssh] Bake in LIBSSH_STATIC

ras0219-msft committed:
[vcpkg] Link pthreads

ras0219-msft committed:
[vcpkg] Improve non-Windows default triplet detection

ras0219-msft committed:
[openssl] Add BSD-generic64

ras0219-msft committed:
[vcpkg] Silence warnings on recent clang

ras0219-msft committed:
[libpq] Fix config writing into source dir

ras0219-msft committed:
[vcpkg] Fix get_executable_path on BSD

ras0219-msft committed:
[libssh] Only build one flavor at a time, don't build examples.

ras0219-msft committed:
[libmysql] Update to 8.0.4

ras0219-msft committed:
[vcpkg] Improve default triplets for non-windows

ras0219-msft committed:
[libssh] Fix static builds for non-windows

ras0219-msft committed:
[libjpeg-turbo] Fix non-windows non-linux

ras0219-msft committed:
[vcpkg-find-acquire-program] Handle PERL and NASM on non-windows

ras0219-msft committed:
[libmysql] Clarify support message

ras0219-msft committed:
[libmariadb] Use vcpkg_from_github() and improve conflict check

ras0219-msft committed:
[vcpkg] Further refactoring inside `build_package()`

ras0219-msft committed:
[vcpkg-cmake-toolchain] Only wrap find_package(Boost) if a boost library is installed.

itsuhane committed:
Add OpenMesh 6.3(without apps) (#3032)

pravic committed:
[curl] update to 7.59.0

ras0219-msft committed:
[jemalloc] Fix corrupted patchfile

ras0219-msft committed:
[zeromq] Update

alexkaratarakis committed:
[opencv] Fixes for contrib

alexkaratarakis committed:
Merge pull request #2906 from cenit/OpenCV_341

[OpenCV] update to 3.4.1

alexkaratarakis committed:
Merge pull request #3033 from pravic/curl

[curl] update to 7.59.0

claudiofantacci committed:
Fix PCL portfile debug/share error (#2996)

Add missing file(REMOVE_RECURSE /debug/share) that was preventing post-build validation to complete successfully.

ras0219-msft committed:
[nlohmann-json] Update to 3.1.2

ras0219-msft committed:
[vcpkg] Create minimal package structure even on failure

SupSuper committed:
Fix boost build errors when vcpkg path has spaces (#3054)

* Fix boost build errors when vcpkg path has spaces

Add missing quotes in boost_serialization path

* [boost-modular-build-helper] Bump version due to change

* [boost-modular-build-helper] Quote mpi as well

Ambrou committed:
Merge branch 'master' of https://github.com/Ambrou/vcpkg

Ambrou committed:
Update gherkin-cp por

ras0219-msft commented:
Merged as part of #3003 -- thanks for the PRs!
